src/scores/categorical/contingency_impl.py

```diff
@@ -211,73 +211,141 @@ def probability_of_detection(self) -> xr.DataArray:
 
         return pod
 
-    def true_positive_rate(self):
+    def true_positive_rate(self) -> xr.DataArray:
         """
-        What proportion of the observed events where correctly forecast?
         Identical to probability_of_detection
```

Comment:
One thing that occured to me when looking at the readthedocs, perhaps we should put these other functions that we refer to in backticks, e.g., `probability_of_detection`

Reply:
I'll raise a new issue to consider that, I'll happily do it in a subsequent round if we want it. I think it's not clear what's best so I'll defer the effort on that.